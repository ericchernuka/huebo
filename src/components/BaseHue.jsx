import React from 'react'
import PropTypes from 'prop-types'
import Label from './Label'

const BaseHue = ({ hue }) => (
  <div className="base-hue-wrapper">
    <Label>Base Hue</Label>
    <div className="color-profile">{`${hue}Â°`}</div>
  </div>
)

BaseHue.propTypes = {
  hue: PropTypes.number.isRequired,
}

export default BaseHue
