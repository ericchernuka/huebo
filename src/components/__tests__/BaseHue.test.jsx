import React from 'react'
import { render } from 'react-testing-library'
import BaseHue from '../BaseHue'

test('renders hue degree', () => {
  const hue = 60
  const { container } = render(<BaseHue hue={hue} />)
  expect(container).toMatchSnapshot()
})
