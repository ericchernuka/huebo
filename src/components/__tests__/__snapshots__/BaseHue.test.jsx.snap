// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders hue degree 1`] = `
<div>
  <div
    class="base-hue-wrapper"
  >
    <h2>
      Base Hue
    </h2>
    <div
      class="color-profile"
      data-testid="base-hue"
    >
      60Â°
    </div>
  </div>
</div>
`;
